package.json

```diff
@@ -34,12 +34,13 @@
     "read-yaml": "^1.1.0",
     "sanitize-filename": "^1.6.1",
     "sass-loader": "^7.1.0",
-    "vuepress": "1.0.0-alpha.47",
+    "vuepress": "1.0.0-beta.1",
     "vuepress-plugin-container": "^2.0.1",
     "vuepress-validate-frontmatter": "^2.0.10",
     "yaml": "^1.5.1"
   },
   "devDependencies": {
+    "colors": "^1.3.3",
```

Comment:
Why this?

Reply:
This is a dependency for the current snippet test stack. 
I don't know how this happened but it was previously registered in the `package-lock.json` but not in the package.json...